src/App.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import axios from 'axios';
-import {BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-import CreateBlogPost from './components/CreateBlogPost'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import BlogPost from './components/BlogPost';
+import NavBar from './components/NavBar';
+import Landing from './components/Landing';
```

Comment:
⛏️ these imports look unused currently

Reply:
You are correct, will remove. Had them because we were planning on adding routing in Week 3 but way to much stuff going on so we pushed it to advanced frontend